nursery/capture-process-snapshot.yml

```diff
@@ -0,0 +1,10 @@
+rule:
+  meta:
+    name: capture process snapshot
```

Comment:
can you think of any way to make this name a little more descriptive? like what kind of snapshot?

perhaps "capture process memory snapshot"?

Reply:
there's multiple information that can be queried, see https://learn.microsoft.com/en-us/windows/win32/api/processsnapshot/ne-processsnapshot-pss_capture_flags, so I suggest data?!